subsys/spm/Kconfig

```diff
@@ -157,6 +157,10 @@ config SPM_NRF_REGULATORS_NS
 	bool "Regulators is Non-Secure"
 	default n
 
+config SPM_NRF_WDT_NS
+	bool "WDT is Non-Secure"
+	default y
```

Comment:
Ah this should probably be `n` as default

Reply:
remove line -> make default n

Reply:
fixed